api/src/app/modules/public-comment/public-comment.service.ts

```diff
@@ -149,11 +209,93 @@ export class PublicCommentService extends DataService<PublicComment, Repository<
     }
 
     const recordIds = _.map(records, 'id');
-    const decryptedColumnCollection = await this.obtainDecryptedColumns(recordIds);
+    const decryptedColumnCollection = await this.obtainDecryptedColumns(
+      recordIds
+    );
     return records.map((r) => {
-      const decryptedPartial = _.find(decryptedColumnCollection, {'id': r.id});
+      const decryptedPartial = _.find(decryptedColumnCollection, { id: r.id });
       Object.assign(r, decryptedPartial);
       return this.convertEntity(r);
     });
   }
+
+  /**
```

Comment:
Sorry I applied the auto formatter by accident. The above are most formatting changes. The real methods are added below. Thanks!